feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -270,15 +270,28 @@ func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
 	}
 
 	if tc.dualSup {
-		if got, want := r.GetVerifyStandby().GetVerifyResponse().GetActivationFailMessage(), ""; got != want {
-			t.Errorf("OS.Verify Standby ActivationFailMessage: got %q, want %q", got, want)
-		}
+		var counter int
+		const maxRetries = 15
+		for i := 1; i < maxRetries; i++ {
+			r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+			if err != nil {
+				t.Fatalf("OS.Verify request failed: %s", err)
+			}
+			if counter == maxRetries {
```

Comment:
This condition will never be true because `counter` is always one less than `i`, and `i` can only ever be one less than `maxRetries`. In general, the for loop idiom should look like this:

```
for i := 1; i <= maxRetries; i++ {
  t.Logf("Attempt %d of %d", i, maxRetries)
  r, err := t.osc.Verify(...)
  ... // check err
  ... // check r
  if got == want {
    break
  }
  if i == maxRetries {
    t.Fatal(...)
  }
  time.Sleep(...)
}
```

Reply:
Fixed the loop with above mentioned idiom in latest commit.